engine/src/main/java/org/terasology/rendering/world/WorldRendererLwjgl.java

```diff
@@ -442,14 +406,14 @@ public void renderWorld(Camera camera) {
         DefaultRenderingProcess.getInstance().beginRenderLightGeometryStencilPass();
         Material program = Assets.getMaterial("engine:prog.simple");
```

Comment:
Thank you for your patience Immortius and explaining me this. I'll look into the ShaderManagerLwjgl and Material classes to understand the details. Indeed I do not forsee the need to change anything in this department at this stage. But I probably need to understand better how it works.